crates/wasmparser/src/features.rs

```diff
@@ -233,19 +233,34 @@ define_wasm_features! {
 }
 
 impl WasmFeatures {
+    /// The feature set associated with the MVP release of WebAssembly (its
+    /// first release).
+    //
+    // Note that the features listed here are the wasmparser-specific built-in
+    // features such as "floats" and "gc-types". These don't actually correspond
+    // to any wasm proposals themselves and instead just gate constructs in
+    // wasm. They're listed here so they otherwise don't have to be listed
+    // below, but for example wasm with `externref` will be rejected due to lack
+    // of `externref` first.
+    #[cfg(feature = "features")]
+    pub const MVP: WasmFeatures = WasmFeatures::FLOATS.union(WasmFeatures::GC_TYPES);
+
     /// The feature set associated with the 1.0 version of the
-    /// WebAssembly specification or the "MVP" feature set.
+    /// WebAssembly specification circa 2017.
+    ///
+    /// <https://webassembly.github.io/spec/versions/core/WebAssembly-1.0.pdf>
     #[cfg(feature = "features")]
-    pub const WASM1: WasmFeatures = WasmFeatures::FLOATS.union(WasmFeatures::GC_TYPES);
+    pub const WASM1: WasmFeatures = WasmFeatures::MVP.union(WasmFeatures::MUTABLE_GLOBAL);
```

Comment:
Surprised that saturating conversions and sign extensions weren't in 1.0, I had thought they were.